javascripts/govuk/shim-links-with-button-role.js

```diff
@@ -1,13 +1,11 @@
 // javascript 'shim' to trigger the click event of element(s)
 // when the space key is pressed.
 //
-// usage instructions:
-// GOVUK.shimLinksWithButtonRole.init();
+// Created since some Assistive Technologies (for example some Screenreaders)
```

Comment:
I thought this was the reason we did this, since some AT are telling users to press space which at the moment does not work?